src/vstart.sh

```diff
@@ -1243,6 +1333,12 @@ osd pool create ec erasure ec-profile
 EOF
 fi
 
+# Ganesha Daemons
+if [ $GANESHA_DAEMON_NUM -gt 0 ]; then
+    echo "Starting ganesha"
+    start_ganesha
+fi
```

Comment:
It might be good to also print out the ports after starting the daemons. NFS clients will need to know which port to talk to since it's not sitting on a standard port.

Reply:
I have updated the patch to print the ports.